frontend/src/modules/scaffold/backboneFormsOverrides.js

```diff
@@ -196,7 +196,7 @@ define([
         dataIndentationChars: '',
         disableNativeSpellChecker: false,
         entities: false,
-        extraAllowedContent: 'span(*)',
+        extraAllowedContent: Origin.constants.ckEditorAllowedContent || 'span(*)',
```

Comment:
Just to say that If the user wanted to allow _no_ extra content `''`, they wouldn't be able because it would fall back to `'span(*)'`.